tests/lib/common/github/issuereporter.ts

```diff
@@ -0,0 +1,212 @@
+});
+
+test(`If there is an error and issue doesn't exists yet, it should create issue`, async (t) => {
+    const sandbox = t.context.sandbox;
+    const Octokit = t.context.Octokit;
+    const octokitSearchIssuesStub = sandbox.stub<OctokitSearch, 'issues'>(Octokit.prototype.search, 'issues').resolves({ data: { items: [] } });
+    const octokitIssuesUpdateSpy = sandbox.spy(Octokit.prototype.issues, 'update');
+    const octokitIssuesCreateStub = sandbox.stub(Octokit.prototype.issues, 'create').resolves();
+    const IssueReporter = loadScript(t.context);
+    const issueReporter = new IssueReporter();
+    const errorMessage = 'Error running webhint';
+
+    await issueReporter.report({
+        configs: [{
+            connector: { name: 'chrome' },
+            hints: { axe: 'error' }
+        }],
+        errorMessage,
+        errorType: 'crash',
+        scan: moment().format('YYYY-MM-DD'),
+        url: 'http://example.com'
+    });
+
+    t.true(octokitSearchIssuesStub.called);
+    t.false(octokitIssuesUpdateSpy.called);
+    t.true(octokitIssuesCreateStub.calledOnce);
+
+    const args = octokitIssuesCreateStub.args[0][0];
+
+    t.true(args.body.includes(errorMessage));
+    t.true(args.body.includes('"axe": "error"'));
+
+    sandbox.restore();
+});
+
+test(`If there is an error and issue exists, it should create a comment`, async (t) => {
+    const sandbox = t.context.sandbox;
+    const Octokit = t.context.Octokit;
+    const octokitSearchIssuesStub = sandbox.stub<OctokitSearch, 'issues'>(Octokit.prototype.search, 'issues').resolves({
+        data: {
+            items: [{
+                labels: [{ name: 'error:timeout' }],
+                number: 1
+            }]
+        }
+    });
+    const octokitIssuesUpdateSpy = sandbox.spy(Octokit.prototype.issues, 'update');
+    const octokitIssuesCreateSpy = sandbox.spy(Octokit.prototype.issues, 'create');
+    const octokitIssuesCreateCommentStub = sandbox.stub(Octokit.prototype.issues, 'createComment').resolves();
+    const IssueReporter = loadScript(t.context);
+    const issueReporter = new IssueReporter();
+    const errorMessage = 'Error running webhint';
+    const scan = moment().format('YYYY-MM-DD');
+
+    try {
+        await issueReporter.report({
+            configs: [{
+                connector: { name: 'chrome' },
+                hints: { hint2: 'warning' }
+            }],
+            errorMessage,
+            errorType: 'timeout',
+            scan,
+            url: 'http://example.com'
+        });
+    } catch (e) {
+        console.log(e);
+    }
+    t.true(octokitSearchIssuesStub.called);
+    t.true(octokitIssuesUpdateSpy.calledOnce);
+    t.false(octokitIssuesCreateSpy.called);
+    t.true(octokitIssuesCreateCommentStub.calledOnce);
+
+    const args = octokitIssuesCreateCommentStub.args[0][0];
+
+    t.true(args.body.includes(errorMessage));
+    t.true(args.body.includes('"hint2": "warning"'));
+    t.is(args.number, undefined);
```

Comment:
check the value of `issue_number` instead of number.
```suggestion
    t.is(args.issue_number, 1);
```